apps/web/src/services/dashboards/components/DashboardWidgetOptionDropdown.vue

```diff
@@ -227,7 +227,7 @@ const initSelectedMenuItems = async (): Promise<SelectDropdownMenuItem[]> => {
         results = initSelectedInInheritCase(inheritOption);
     // 2) non-inherit case
     } else {
-        const selected: Array<ConsoleFilter|string>|string|undefined = get(widgetFormGetters.updatedWidgetInfo?.widget_options, props.propertyName)
+        const selected: SelectDropdownMenuItem[]|string[]|string|undefined = get(widgetFormGetters.updatedWidgetInfo?.widget_options, props.propertyName)
```

Comment:
issue: has widget_options.filter's type changed? I think we didn't converted it yet. this type is NOT `SelectDropdownMenuItem[]|string[]|string|undefined`